### PR TITLE
Makes template list Responsive

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This PR addresses #4247.  It add responsiveness to the Template's List.  

##### ISSUE TYPE
- Bug

##### COMPONENT NAME
 - UI

##### AWX VERSION
```
7.0.0
```

##### ADDITIONAL INFORMATION
templatelist

### CODE REVIEW

File: awx/ui_next/src/components/DataListCheck/DataListCheck.jsx [hunk 1]
```diff
@@ -2,8 +2,13 @@ import { DataListCheck as PFDataListCheck } from '@patternfly/react-core';
 import styled from 'styled-components';
 
 export default styled(PFDataListCheck)`
-  .pf-c-data-list__check {
-    display: flex;
-    align-items: center;
+  padding-top: 18px;
+  @media screen and (min-width: 768px) {
```
Comment (jakemcdermott) [inline awx/ui_next/src/components/DataListCheck/DataListCheck.jsx:6]: Why 768?

Comment (jakemcdermott) [reply awx/ui_next/src/components/DataListCheck/DataListCheck.jsx:6]: nevermind

File: awx/ui_next/src/screens/Template/TemplateList/TemplateListItem.jsx [hunk 2]
```diff
@@ -27,6 +27,32 @@ const StyledButton = styled(PFButton)`
     color: white;
   }
 `;
+const DataListItemCells = styled(PFDataListItemCells)`
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Template/TemplateList/TemplateListItem.jsx:30]: Do we need these on the other lists as well?

**Review by jakemcdermott — APPROVED** [review]
:+1: lets merge

File: awx/ui_next/src/screens/Template/TemplateList/TemplateListItem.jsx (hunk 2 above)
Comment (AlexSCorey) [reply awx/ui_next/src/screens/Template/TemplateList/TemplateListItem.jsx:30]: @jakemcdermott Yeah we probably _could_ use those styling in other lists.  I could make them props that are passed into our `DatalistCell` component.  That way we have the choice to use them if we want to.

**Pull request merged**